gammapy/modeling/models/spatial.py

```diff
@@ -518,7 +513,7 @@ def evaluation_radius(self):
 
     def to_region(self, **kwargs):
         """Model outline (`~regions.EllipseSkyRegion`)."""
-        minor_axis = Angle(self.r_0.quantity * (1 - self.e.quantity))
+        minor_axis = Angle(self.r_0.quantity * np.sqrt(1 - self.e.quantity ** 2))
```

Comment:
If the previous implementation was incorrect, can you please add a regression test, so that change is covered by a test?

Reply:
added